autoware_launch/launch/autoware.launch.xml

```diff
@@ -14,6 +16,12 @@
     <arg name="vehicle_model" value="$(var vehicle_model)"/>
   </include>
 
+  <!-- Pointcloud container -->
+  <include file="$(find-pkg-share autoware_launch)/launch/pointcloud_container.launch.py" if="$(var use_pointcloud_container)">
+    <arg name="use_multithread" value="false" />
```

Comment:
```suggestion
    <arg name="use_multithread" value="true" />
```

Reply:
fix in https://github.com/tier4/AutowareArchitectureProposal_launcher/pull/147/commits/8b4e06ec17a57f38cdf73cd1b444585dcca3843a